bundles/org.eclipse.ui.forms/src/org/eclipse/ui/forms/widgets/ExpandableComposite.java

```diff
@@ -933,6 +933,16 @@ public void removeExpansionListener(IExpansionListener listener) {
 		listeners.remove(listener);
 	}
 
+	/**
+	 * Gets the expansion listeners.
+	 *
+	 * @param listener
+	 *            the listener to remove
+	 */
+	public ListenerList<IExpansionListener> getExpansionListeners() {
+		return listeners;
```

Comment:
Maybe you can copy the listeners here into an array instead of return the (modifiable) list.